src/constants/settings.ts

```diff
@@ -0,0 +1,37 @@
+import StellarSdk from "stellar-sdk";
+import { store } from "App";
+import { NetworkType } from "constants/types.d";
+
+interface NetworkItemConfig {
+  url: string;
+  network: string;
+  stellarExpertTxUrl: string;
+}
+
+interface NetworkConfig {
+  testnet: NetworkItemConfig;
+  public: NetworkItemConfig;
+}
+
+export const networkConfig: NetworkConfig = {
+  testnet: {
+    url: "https://horizon-testnet.stellar.org",
+    network: StellarSdk.Networks.TESTNET,
+    stellarExpertTxUrl: "https://stellar.expert/explorer/testnet/tx/",
+  },
+  public: {
+    url: "https://horizon.stellar.org",
+    network: StellarSdk.Networks.PUBLIC,
+    stellarExpertTxUrl: "https://stellar.expert/explorer/public/tx/",
+  },
+};
+
+const isTestnet = () => {
+  const { settings } = store.getState();
+  return settings.isTestnet;
+};
+
+export const getNetworkConfig = () => {
+  const network = isTestnet() ? NetworkType.TESTNET : NetworkType.PUBLIC;
+  return networkConfig[network];
+};
```

Comment:
I'm not sure I like these two functions for two reasons:

- They're not constants
- They reach directly into the store (which might cause issues like circular references)

Is this a problem for you? Can you think of a solution?

Reply:
No problem at all, I'll move it to helpers and will pass the network flag from redux where it is called. I thought this approach would save those extra imports.